pubspec.yaml

```diff
@@ -7,7 +7,7 @@ name: stack_trace
 #
 # When the major version is upgraded, you *must* update that version constraint
 # in pub to stay in sync with this.
-version: 1.8.1
+version: 1.8.2-dev
```

Comment:
We're getting ready to publish. Remove the `-dev`

Reply:
done.